docs/architecture/tablets.rst

```diff
@@ -0,0 +1,92 @@
+=========================================
+Data Distribution  with Tablets
+=========================================
+
+A ScyllaDB cluster is a group of interconnected nodes - on-premises servers or 
+virtual servers (public cloud instances). The data of the entire cluster has to
```

Comment:
```suggestion
A ScyllaDB cluster is a group of interconnected nodes. The data of the entire cluster has to
```

Reply:
I agree with the spirit if this change since on-prem vs. cloud has nothing to do with tablets and it seems like marketing verbiage put of place in a technical doc